stdlib/src/collections/dict.mojo

```diff
@@ -612,6 +612,30 @@ struct Dict[K: KeyElement, V: CollectionElement](
             return ev.value()[].value
         return None
 
+    fn get(self, key: K) -> Optional[V]:
+        """Get a value from the dictionary by key.
+
+        Args:
+            key: The key to search for in the dictionary.
+
+        Returns:
+            An optional value containing a copy of the value if it was present,
+            otherwise an empty Optional.
+        """
+        return self.find(key)
+
+    fn get(self, key: K, default: V) -> V:
```

Comment:
`self.find(key)` has already unit tests. But `get(self, key: K, default: V) -> V` doesn't have unit tests yet. Maybe you can add a few unit tests for this function in https://github.com/modularml/mojo/blob/nightly/stdlib/test/collections/test_dict.mojo ?